### PR TITLE
Initiate bucket docs: Your bucket name

### LINKED ISSUE #80: Docs: Adding files to S3
write docs with instructions to add files to S3

### LINKED ISSUE #81: Docs: Finding your bucket on S3
Write instructions for users to find their bucket on S3

### PR DESCRIPTION
**Pull request by sjspielman**
### Which issue does this address?

Closes #80 
Closes #81

### Briefly describe the scope of the added docs file, including whether you link out to any external docs.

This PR starts off the S3 bucket docs with:

- a bucket page
- how to find your bucket name
- tell contributors they will have to provide their bucket name in PRs
- a little advanced subsection that they can store their bucket name in an environment variable

Originally, we had planned for three separate docs pages for #80, #81, and #82. As I started to write this, I thought it would make more sense to just have a single page with all this information, since each section is fairly short! Let me know if you disagree with this decision. 

### Will you need additional visual aids for these docs?

No

### Any other comments for reviewers?

No

### Author checklist

- [x] The docs page has been added to the `nav` section in `mkdocs.yml`
- [x] The docs adhere to the style guide [**structure directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)
- [x] The docs adhere to the style guide [**language directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)
- [x] The docs adhere to the style guide [**tone directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#tone)
- [x] The docs follow additional contributing instructions in [the docs contributing file](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md)

### Reviewer checklist

Please refer to the [docs style guide](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md) and the [docs contributing guide](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md) while reviewing this pull request.

- [ ] Do the docs adhere to the style guide [**structure directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)?
- [ ] Do the docs adhere to the style guide [**language directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)?
- [ ] Do the docs adhere to the style guide [**tone directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#tone)?
- [ ] Do the docs follow additional contributing instructions in [the docs contributing file](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md)?

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -132,6 +132,7 @@ nav:
       - software-platforms/lsfr/index.md
     - Amazon S3:
```
Comment (jaclyn-taroni) [inline mkdocs.yml:133]: In my opinion, this needs to be rearranged a bit. When it got stubbed in, it was assumed that the header would be S3, but LfR is also AWS! The AWS index page talks about signing up, which is pretty important!

Comment (sjspielman) [reply mkdocs.yml:133]: @dvenprasad I think it's probably ok to just remove the `lsfr` directory altogether, and just put all those pages into the `aws` directory?

Comment (sjspielman) [reply mkdocs.yml:133]: The 25th iteration of our IA has landed 😂 #61

Comment (jaclyn-taroni) [reply mkdocs.yml:133]: Okay maybe we take a pass at rearranging all at once? The relative link headache will be real.

File: docs/software-platforms/aws/working-with-s3-buckets.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+This page covers some information you'll need for working with your S3 bucket.
+
+!!! note "Still need to join OpenScPCA?"
+    Start the process of becoming a contributor by [filing out project intake form](https://share.hsforms.com/1MlLtkGYSQa6j23HY_0fKaw336z0).
+
+
+## Finding your bucket name
+
+Once you have been accepted as a new OpenScPCA contributor, you will receive an account number specific to you.
+Your S3 bucket name is simply `researcher-{account number}-us-east-2`.
```
Comment (jashapiro) [inline docs/software-platforms/aws/working-with-s3-buckets.md:16]: It isn't simple, sadly.
```suggestion
Your S3 bucket name is `researcher-{account number}-us-east-2`.
```

File: docs/software-platforms/aws/working-with-s3-buckets.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+> These instructions are mostly applicable to advanced users, but we're happy to help you set this up if you need assistance!
+
+Optionally, you may wish to save your bucket name as an environment variable in your shell profile file (e.g., `~/.bashrc` for Bash users and `~/.zshrc` for Z shell users).
+
+We have set up some OpenScPCA scripts to look for the environment variable `OPENSCPCA_RESULTS_BUCKET`, so if you define this in your profile, you'll have an easier time using those scripts!
+
+For this, add this line (with `{account number}` replaced with your account number) to your profile file:
+```sh
+export OPENSCPCA_RESULTS_BUCKET=researcher-{account number}-us-east-2
+```
```
Comment (jashapiro) [inline docs/software-platforms/aws/working-with-s3-buckets.md:36]:
````suggestion
To use this, add this line (with `{account number}` replaced with your account number) to your profile file:

```sh
export OPENSCPCA_RESULTS_BUCKET=researcher-{account number}-us-east-2
```
````

File: docs/software-platforms/aws/working-with-s3-buckets.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`) to refer to your bucket name when running commands in terminal.
```
Comment (jashapiro) [inline docs/software-platforms/aws/working-with-s3-buckets.md:38]:
```suggestion
You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`) to refer to your bucket name when running commands in the terminal or in shell scripts.
```

Comment (jashapiro) [reply docs/software-platforms/aws/working-with-s3-buckets.md:38]: I'm not actually sure I had this use case in mind, but it seems fine to mention it.

**Review by jashapiro — COMMENTED** [review]
LGTM, but one does not simply include an account number.

**Comment by sjspielman** [comment]
@jashapiro any other feedback here? Note that a few stacked PRs have been merged into this one.

File: README.md [hunk 5]
```diff
@@ -64,12 +64,11 @@ To set up, you will need a Git client, Miniconda, R or Python, and an editor of
 Data is publicly available from the [ScPCA Portal](https://scpca.alexslemonade.org/) and from an AWS S3 bucket for OpenScPCA project contributors.
 We also provide a way to download smaller, simulated data files for you to play with.
 
-Please reference [Getting access to Data](https://openscpca.readthedocs.io/en/latest/getting-started/accessing-resources/getting-access-to-data/) for more details.
+Please refer to [Getting access to Data](https://openscpca.readthedocs.io/en/latest/getting-started/accessing-resources/getting-access-to-data/) for more details.
```
Comment (jashapiro) [inline README.md:67]:
```suggestion
Please refer to [Getting Access to Data](https://openscpca.readthedocs.io/en/latest/getting-started/accessing-resources/getting-access-to-data/) for more details.
```

File: docs/software-platforms/aws/working-with-s3-buckets.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    --bucket {name of your researcher bucket} \
+    --dryrun
+```
+
+If everything looks correct to you when running with `--dryrun`, you can then proceed to run the script without this flag and actually perform the syncing.
+
+### Additional script flags
+
+By default, if there are any result or plot files that exist on S3 but that you have locally deleted, this script will _not delete_ those files from S3.
+To override this behavior and delete these files from S3 as well (but only for the specified analysis module), use the `--destructive-sync` flag:
```
Comment (jashapiro) [inline docs/software-platforms/aws/working-with-s3-buckets.md:82]: Space for the markdown gods.
```suggestion
To override this behavior and delete these files from S3 as well (but only for the specified analysis module), use the `--destructive-sync` flag:

```

**Review by jashapiro — APPROVED** [review]
LGTM, with two tiny suggestions

**Pull request merged**